Examples/Swift/CustomNavigationUI.swift

```diff
@@ -0,0 +1,170 @@
+import UIKit
+import MapboxCoreNavigation
+import Mapbox
+import CoreLocation
+import AVFoundation
+import MapboxDirections
+
+class CustomNavigationUI: UIViewController, MGLMapViewDelegate, AVSpeechSynthesizerDelegate {
```

Comment:
This class name should end with `ViewController`, however, `CustomNavigationViewController` sounds like it's a subclass of `NavigationViewController` which is misleading.